pyproject.toml

```diff
@@ -25,6 +25,7 @@ pytest = { version = ">=5.0", optional = true }
 pytest-cov = { version = "2.10.1", optional = true }
 deepdiff = { version = ">=6.2.3", optional = true }
 neptune = { version = ">=1.0.0", optional = true }
+cffi = { version = "*", optional = true}
```

Comment:
You have to add i under `dev` extras (line 37) if you want to use `pip install -e .[dev]`.